ml-proto/spec/eval.ml

```diff
@@ -88,6 +88,9 @@ let numerics_error at = function
       error at "runtime: invalid conversion to integer"
   | exn -> raise exn
 
+let some_memory c =
+  match c.module_.memory with Some m -> m | _ -> assert false
```

Comment:
Not strong reason: just because the source info clutters a bit and while it's generally worth it because it helps us when developing, the `Crash`es should never happen, even when developing.  That's why I liked `assert`.  Since impl=spec (sortof), I do wonder if we need to discriminate bugs in impl and bugs in semantics.